<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projectWindow</class>
 <widget class="QMainWindow" name="projectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReadySet Volunteer - Project Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>771</width>
      <height>192</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Project</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Supervisor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionEdit_Project"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionEdit_Project">
   <property name="text">
    <string>Edit Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
